client/src/test/java/uk/co/blackpepper/bowman/LinkedResourceMethodHandlerTest.java

```diff
@@ -0,0 +1,88 @@
+package uk.co.blackpepper.bowman;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+import org.junit.Test;
+import org.springframework.hateoas.Resource;
+
+import uk.co.blackpepper.bowman.annotation.LinkedResource;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.core.IsNull.notNullValue;
+import static org.junit.Assert.assertThat;
+import static org.springframework.util.ReflectionUtils.findMethod;
+
+public class LinkedResourceMethodHandlerTest {
+
+	private final ResourceContent resourceContent = new ResourceContent();
+	private final LinkedResourceMethodHandler handler = createHandler();
+
+	@Test
```

Comment:
I think these `supports*` tests are a bit misleading - the implementation doesn't care whether it's a getter that's annotated for instance, which might be confusing to the reader. I think your permutations here are:

1. `@LinkedResource` on method
1. `@LinkedResource` on setter's corresponding getter
1. No `@LinkedResource`